tests/__init__.py

```diff
@@ -27,9 +27,9 @@
 from ratbag_emu.protocol.base import MouseData
 
 
-class Client(object):
-    def __init__(self, url='http://localhost:8080'):
-        self.url = url
+class RatbagemuClient(object):
```

Comment:
the commit message doesn't match the patch, it's a bit confusing.

Reply:
Which part exactly is confusing? In the libratbag test suite (https://github.com/libratbag/libratbag/pull/813) we have a `RatbagemuClient` class, I just added it here. Maybe I can add a better commit description.

Reply:
oh, in that case "copy RatbagEmuClient() ..." in the commit message please. "use" would imply you're actually importing and using one from libratbag.